src/solve/opaque-types.md

```diff
@@ -116,4 +117,4 @@ fn go(i: usize) -> impl Future<Output = ()> + Send + 'static {
         }
     }
 }
-```
\ No newline at end of file
+```
```

Comment:
i have no idea what's going on there

Reply:
It's the new line at the end of the file. My editor (VS Code) is set up the same and it seems the some `.md` files have the trailing new line and some do not. That said, I guess the change is fine.